features/dd-sdk-android-trace/src/main/kotlin/com/datadog/android/trace/internal/handlers/AndroidSpanLogsHandler.kt

```diff
@@ -67,6 +67,8 @@ internal class AndroidSpanLogsHandler(
         timestampMicroseconds: Long? = null
     ) {
         val logsFeature = sdkCore.getFeature(Feature.LOGS_FEATURE_NAME)
+        val spanLogSeverity = fields["severity"] as? Int
```

Comment:
we need to remove this from the attributes, since it is a technical property which shouldn't end in the list of the user attributes

```suggestion
        val spanLogSeverity = fields.remove("severity") as? Int
```

also we need to rename it from `severity`

Reply:
thanks for spotting this

1. renamed to "status"
2. now I remove it from the fields